src/config/profileConfig.js

```diff
@@ -91,15 +91,15 @@ const buildConfig = () => {
             },
             {
               name: 'evaluateMeasure',
-              route: '/:id/$evaluate-measure',
+              route: '/:id/$evaluate',
               method: 'GET',
-              reference: 'https://www.hl7.org/fhir/measure-operation-evaluate-measure.html'
+              reference: 'https://www.hl7.org/fhir/measure-operation-evaluate.html'
```

Comment:
@lmd59 should we link to the R4 measure-operation-evaluate-measure (https://hl7.org/fhir/R4/measure-operation-evaluate-measure.html) since we still have it return a Measure Report? Or should we Link to the DEQM evaluate operation (https://build.fhir.org/ig/HL7/davinci-deqm/OperationDefinition-evaluate.html) since that is what we are transitioning to?